### ISSUE TITLE
Backend false generated links

### ISSUE THREAD

**Issue by rmilenov** [in2code-de/luxletter#200]
hey there, I'm not quite sure what the problem is, but when I access the Luxletter tab in the back end I get the dashboard page loaded and 
I#m not able to change it. No Problem in typo3 log but the link is clearly not right. 

https://webserver.rado/typo3/index.php?route=%2Fmodule%2Flux%2FLuxletterLuxletter&token=17b0ca4a2f10aeff35c1744bd76c3d6bab7777c5&tx__%5Baction%5D=list&tx__%5Bcontroller%5D=Newsletter

typo3 10.4.37, Luxletter: 17.8.0

Did someone experienced such behaviour ?

**Comment by einpraegsam** [comment]
Hi, maybe this is a bug in the old version of LUXletter. Meanwhile we're on version 24. The old version 17 is not supported any more.

**Comment by rmilenov** [comment]
Last available for typo3 10.4 released 10 days ago...

**Comment by einpraegsam** [comment]
Yes, some payed features or bugfixes are also backported - that's correct
